src/control/controlframeposproxy.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "audio/frame.h"
+#include "control/controlproxy.h"
+
+/// Special kind of `ControlProxy` that is used for storing frame positions.
```

Comment:
We should name the design pattern that is used here, probably *adapter* or *decorator* (didn't check which one fits).